packages/vitest-pool-workers/src/worker/lib/mlly.ts

```diff
@@ -16,7 +16,7 @@ export function interopDefault(sourceModule: any): any {
 	for (const key in sourceModule) {
 		if (key === "default") {
 			try {
-				if (!(key in newModule)) {
+				if (!(key in (newModule as Record<string, unknown>))) {
```

Comment:
I think you can avoid the type assertions in this file by moving them to the `isObject` function above:

```
function isObject(value: unknown): value is Record<string, unknown> {
	return value !== null && typeof value === "object";
}
```

Then add the same check for the `default` object:


```
if (
		!isObject(sourceModule) ||
		!("default" in sourceModule) ||
		!isObject(sourceModule.default)
	) {
		return sourceModule;
	}
```

Reply:
awesome, thank you!